samples/sample-crashlytics/shared/shared.podspec

```diff
@@ -32,7 +32,7 @@ Pod::Spec.new do |spec|
                 fi
                 set -ev
                 REPO_ROOT="$PODS_TARGET_SRCROOT"
-                "$REPO_ROOT/../gradlew" -p "$REPO_ROOT" $KOTLIN_PROJECT_PATH:syncFramework \
+                "$REPO_ROOT/../../../../../../../private/var/folders/vh/lqnbj11s3j53773y2g1kyr6c0000gn/T/wrap4838loc/gradlew" -p "$REPO_ROOT" $KOTLIN_PROJECT_PATH:syncFramework \
```

Comment:
Not sure why these podspec changes happened but will revert

Reply:
Weird

Reply:
This is really odd, the more I think about it. I think you *might* want to try to figure out what's happening here. To less experienced KMP devs, this might shut down the team. I guess if the podspec is automatically regenerated it won't be a huge deal, but still confusing.